lib/crowbar/init/application.rb

```diff
@@ -14,6 +14,8 @@
 # limitations under the License.
 #
 
+require "/opt/dell/crowbar_framework/lib/crowbar/upgrade_status"
```

Comment:
It's not wrong but also no good. It slows down the loading speed and if we need to load more shared libs we have duplicated code. 

Can you check to register the LOAD_PATH globally when the package gets installed?